rules/linux/discovery_pspy_process_monitoring_detected.toml

```diff
@@ -0,0 +1,71 @@
+[metadata]
+creation_date = "2023/07/20"
+integration = ["auditd_manager"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/07/20"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule leverages auditd to monitor for processes scanning different processes within the /proc directory using the 
+openat syscall. This is a strong indication for the usage of the pspy utility. Attackers may leverage the pspy process 
+monitoring utility to monitor system processes without requiring root permissions, in order to find potential privilege 
+escalation vectors.
+"""
+from = "now-9m"
+index = ["logs-auditd_manager.auditd-*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Potential Pspy Process Monitoring Detected"
+note = """## Setup
+This rule requires the use of the `auditd_manager` integration. `Auditd_manager` is a tool designed to simplify and enhance the management of the audit subsystem in Linux systems. It provides a user-friendly interface and automation capabilities for configuring and monitoring system auditing through the auditd daemon. With `auditd_manager`, administrators can easily define audit rules, track system events, and generate comprehensive audit reports, improving overall security and compliance in the system. The following steps should be executed in order to install and deploy `auditd_manager` on a Linux system. 
+
+```
+Kibana -->
+Management -->
+Integrations -->
+Auditd Manager -->
+Add Auditd Manager
```

Comment:
When the rule is built, `related_integrations` is added to the rule where `auditd_manager` and a compatible version is added. This is then used by the Kibana UI to show what integration needs installed and has a hyperlink to the integrations page for auditd manager. 

Doesn't hurt to include it in setup as well, but wanted to share this.

Reply:
Thanks for sharing, will do an overhaul of all of the setup guides I wrote for auditd_manager in a seperate PR when the docs team has looked at it as well. Will add this to my notes.